extension/js/common/browser/ui.ts

```diff
@@ -205,13 +204,17 @@ export class Ui {
       });
       return typeof dismiss === 'undefined';
     },
-    page: async (htmlUrl: string): Promise<void> => {
-      const html = await (await fetch(htmlUrl)).text();
+    page: async (htmlUrl: string, replaceNewlines = false): Promise<void> => {
+      let html = await (await fetch(htmlUrl)).text();
```

Comment:
This is interesting. You pre-fetch the html from the file, and render it directly? That is, an iframe is not used?

Reply:
correct

Reply:
Now I see there is a separate `iframe` method. Now I understand, you choose depending if it's static content or not. Makes sense.